iati/fetch/tests/test_dataset.py

```diff
@@ -4,6 +4,8 @@
 import pkg_resources
 import responses
 
+MOCK_REGISTRY_URL = 'http://test.com/{0}'
```

Comment:
https://tools.ietf.org/html/rfc2606 defines domains to use for purposes such as this